ellama.el

```diff
@@ -93,12 +97,12 @@
 (defcustom ellama-keymap-prefix "C-c e"
   "Key sequence for Ellama Commands."
   :type 'string
-  :group 'tools)
+  :group 'ellama)
 
 (defcustom ellama-ollama-binary (executable-find "ollama")
   "Path to ollama binary."
   :type 'string
-  :group 'tools)
+  :group 'ellama)
 
 (defcustom ellama-auto-scroll nil
```

Comment:
Why don't changed group here?

Reply:
Ah, I pulled this patch out of my next patch and added that change to the wrong patch. But I missed the kyemap case in both PRs.

Fixed now.